src/components/Atoms/Select/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { Checkbox as MuiCheckBox } from '@mui/material'
+import React from 'react'
+import { styled } from '@mui/material'
+const label = { inputProps: { 'aria-label': 'Checkbox demo' } }
```

Comment:
Add proper naming instead of "Checkbox Demo". 
- "Check Box" for default.
- if label is given as prop give that value.